Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -563,6 +563,12 @@ bool CStaticFunctionDefinitions::GetElementBoundingBox(CClientEntity& Entity, CV
             pModelInfo = g_pGame->GetModelInfo(Object.GetModel());
             break;
         }
+        case CCLIENTBUILDING:
+        {
+            CClientBuilding& building = static_cast<CClientBuilding&>(Entity);
+            pModelInfo = g_pGame->GetModelInfo(building.GetModel());
+        }
```

Comment:
Although this is the last case, better to add a trailing `brake` to it.

Reply:
Thanks you!